src/components/vuestic-components/va-badge/VaBadge.vue

```diff
@@ -0,0 +1,252 @@
+      required: false,
+    },
+    color: {
+      type: String,
+      default: 'success',
+      required: false,
+    },
+    textColor: {
+      type: String,
+      default: '',
+      required: false,
+    },
+    label: {
+      type: [String, Number],
+      default: '',
+      required: false,
+    },
+    overlap: {
+      type: Boolean,
+      default: false,
+      required: false,
+    },
+    multiline: {
+      type: Boolean,
+      default: false,
+      required: false,
+    },
+    visibleEmpty: {
+      type: Boolean,
+      default: false,
+      required: false,
+    },
+    dot: {
+      type: Boolean,
+      default: false,
+      required: false,
+    },
+    transparent: {
+      type: [String, Number, null],
+      default: null,
+      required: false,
+    },
+    left: {
+      type: Boolean,
+      default: false,
+      required: false,
+    },
+    bottom: {
+      type: Boolean,
+      default: false,
+      required: false,
+    },
+    approximateNumber: {
+      type: Boolean,
+      default: false,
+      required: false,
+    },
+  },
+  computed: {
+    isEmpty () {
+      if (this.label || this.visibleEmpty || this.dot || this.$slots.badge) {
+        return false
+      }
+
+      return true
+    },
+    isFloating () {
+      return this.$slots.default || this.dot
+    },
+    formattedLabel () {
+      const isApproximateNumber = () => this.approximateNumber && this.label && !isNaN(Number(this.label))
+
+      if (isApproximateNumber()) {
+        return approximateNumber(Number(this.label))
+      }
+
+      return this.label
+    },
+    badgeClass () {
+      return {
+        'va-badge--empty': this.isEmpty,
+        'va-badge--dot': this.dot,
+        'va-badge--multiline': this.multiline,
+        'va-badge--floating': this.isFloating,
+        'va-badge--left': this.left,
+        'va-badge--bottom': this.bottom,
+        'va-badge--overlap': this.overlap,
+      }
+    },
+    badgeStyle () {
+      const computedStyles = {
+        borderColor: this.colorComputed,
+        backgroundColor: this.colorComputed,
+      }
+
+      if (this.outline) {
+        computedStyles.color = this.colorComputed
+        computedStyles.backgroundColor = 'transparent'
+      }
+
+      if (this.textColor) {
+        computedStyles.color = this.textColor
+      }
+
+      if (this.transparent !== null && this.transparent !== undefined) {
+        // TODO: need to add lodash isNil
+        if (this.transparent === '') {
+          // if transparent is equal empty string
+          computedStyles.opacity = 0.5
+        } else {
+          const opacityNumber = Number(this.transparent)
+
+          computedStyles.opacity = isNaN(opacityNumber) ? 1 : opacityNumber
+        }
+      }
+
+      return computedStyles
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+@import '../../vuestic-sass/resources/resources';
+
+.va-badge {
+  display: inline-flex;
+  position: relative;
+}
+
+.va-badge__badge {
```

Comment:
`.va-badge__content-wrapper` maybe?

Reply:
No, it is correct. For example, vuetify have a similar realization badge component
https://vuetifyjs.com/en/components/badges#badges

Reply:
If vuetify did something wrongly doesn't mean we have to do the same.

badge.badge - makes no sense to me whatsoever.